### PR TITLE
Reintroduce badges

### LINKED ISSUE #21: Re-introduce badges component

### PR DESCRIPTION
**Pull request by danielmatthew**
Back by popular demand…

Closes #21

### CODE REVIEW

File: dist/app.min.css [hunk 1]
```diff
@@ -3349,6 +3349,42 @@ tbody.collapse.in {
...
+  font-size: 11px;
+  font-weight: bold;
+  color: #fff;
+  line-height: 1;
+  vertical-align: middle;
+  white-space: nowrap;
+  text-align: center;
+  background-color: #8c8c8c;
+  border-radius: 10px; }
+  .badge:empty {
```
Comment (camillef) [inline dist/app.min.css]: badges inside badges?! #badgeception

File: dist/app.min.css [hunk 2]
```diff
@@ -3349,6 +3349,42 @@ tbody.collapse.in {
...
+    display: none; }
+  .btn .badge {
+    position: relative;
+    top: -1px; }
+  .btn-xs .badge, .btn-group-xs > .btn .badge, .btn-group-xs > .btn .badge {
```
Comment (camillef) [inline dist/app.min.css]: You've got `.btn-group-xs > .btn .badge` twice here

**Comment by camillef** [comment]
Apart from comment 👍

File: dist/app.min.css (hunk 1 above)
Comment (danielmatthew) [inline dist/app.min.css]: I think it's just the formatting? Closing brace on 3364.

File: dist/app.min.css (hunk 2 above)
Comment (danielmatthew) [inline dist/app.min.css]: True, though this is the generated code. I'll see if I can suss why...

**Comment by pads** [comment]
I get what you mean by badges now I think @danielmatthew. It highlights things like a notifications count? I imagine this is of use to the Talis player etc.

**Pull request merged**